### ISSUE TITLE
RTL (right-to-left) text is aligned right with zero margin 

### ISSUE THREAD

**Issue by kbdragon** [kepano/obsidian-minimal#414]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

5.3.2 and earlier

### Describe the bug

Obsidian has no native RTL display. The RTL Support (https://github.com/esm7/obsidian-rtl) plugin leverage CodeMirror capabilities to provide this ability per document. 
With the minimal theme enabled, any document written in Hebrew on my machine is aligned correctly but with zero margin between app edge and the text. 
Setting the default theme solves the issue.

Obsidian 0.15.9 with Minimal Theme 5.3.2 and RTL Support 0.3.0.
![image](https://user-images.githubusercontent.com/113253861/189500915-27b807ef-1fe8-4ba2-ac78-b3836823eee8.png)

### Debug info

SYSTEM INFO:
	Obsidian version: v0.15.9
	Installer version: v0.15.9
	Operating system: #1 SMP PREEMPT Sat Aug 20 06:57:36 UTC 2022 5.15.61-217.current
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 36
	Plugins enabled: 24
		1: Admonition v9.0.5
		2: Better Word Count v0.8.1
		3: Book Search v0.5.7
		4: Calendar v1.5.10
		5: Citations v0.4.4
		6: cMenu v1.1.2
		7: Contextual Typography v2.2.4
		8: Dataview v0.5.43
		9: Graph Analysis v0.15.4
		10: Hider v1.1.1
		11: Kanban v1.3.17
		12: Longform v1.1.0
		13: Minimal Theme Settings v5.3.2
		14: Natural Language Dates v0.6.1
		15: Obsidian Charts v3.6.3
		16: Obsidian42 - BRAT v0.6.35
		17: Periodic Notes v0.0.17
		18: QuickAdd v0.5.5
		19: Reading Time v1.1.1
		20: RTL Support v0.3.0
		21: Smart Typography v1.0.18
		22: Sortable v0.2.6
		23: Typewriter Scroll v0.2.2
		24: Zotero Integration v2.2.24

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by AmmarCodes** [comment]
I'm using this css snippet to handle RTL (it flips the margin definition):

```css
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-content>.image-embed, 
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-content>img:not([width]),
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-embed-block.cm-callout>.callout,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-html-embed,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-line,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-line.HyperMD-codeblock,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .cm-line.HyperMD-quote,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .embedded-backlinks,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width .internal-embed,
.is-rtl.markdown-source-view.mod-cm6.is-readable-line-width table {
  margin-left: auto !important;
  margin-right: max(calc(50% + var(--folding-offset) - var(--line-width-adaptive)/ 2),calc(50% + var(--folding-offset) - var(--max-width)/ 2))!important;
}
```

**Comment by kepano** [comment]
This should be fixed in 7.0+
